tests/robot-tests/tests/libs/admin-common.robot

```diff
@@ -320,8 +320,10 @@ user creates methodology amendment for publication
     ...    ${topic}=%{TEST_TOPIC_NAME}
     ${accordion}=    user opens publication on the admin dashboard    ${publication}
     user opens details dropdown    ${methodology_title}    ${accordion}
+    user waits until button is enabled    Amend methodology
```

Comment:
I'm not sure a `user waits until button is enabled` will change anything here, because `user clicks button` already checks that the button is enabled - `user clicks button` calls `user clicks element` which contains a `wait until element is enabled`.

Same issue with other times you've added `user waits until button is enabled` in this PR.

I'd be interested in seeing the failure you're trying to address here.

Reply:
It was a selenium exception. I'm not sure why this fixed it